pkgs/development/libraries/glibc/common.nix

```diff
@@ -115,6 +115,8 @@ stdenv.mkDerivation ({
       ./fix-x64-abi.patch
       ./2.30-cve-2020-1752.patch
       ./2.31-cve-2020-10029.patch
+      ./2.31-cve-2020-6096.0.patch
+      ./2.31-cve-2020-6096.1.patch
```

Comment:
Any particular reason why we don't use fetchurl to grab those patches?

Reply:
bootstrap can't use fetchpatch and fetchurl might not be stable.

Reply:
Yes, instructions in common.nix states to download them directly as they're a part of the bootstrap process:

```
# Note: this package is used for bootstrapping fetchurl, and thus
# cannot use fetchpatch! All mutable patches (generated by GitHub or
# cgit) that are needed here should be included directly in Nixpkgs as
# files.
```
https://github.com/NixOS/nixpkgs/blob/31487784f9ae66e715b2df1e7727119890d05a39/pkgs/development/libraries/glibc/common.nix#L20-L23